kubefed/installing-kubefed.adoc

```diff
@@ -0,0 +1,31 @@
+[id="installing-kubefed"]
+= Installing OpenShift KubeFed
+include::modules/common-attributes.adoc[]
+include::modules/kubefed-document-attributes.adoc[]
+:context: installing-kubefed
+
+To get started with {KubeFedProductName}, you must create the `openshift-federation-system` namespace, install the Operator in this namespace, and then create the required custom resources (CRs) for your deployment.
+
+NOTE: {KubeFedProductName} can be deployed as either a namespace-scoped or a cluster-scoped deployment, but not both, as this will cause conflicts when both deployments attempt to manage the same set of resources.
+
+== Creating the namespace
+
+You must create the `openshift-federation-system` namespace before installing any other {KubeFedProductName} components by using the following command:
+
+----
+oc create ns openshift-federation-system
+----
+
+== Installing the {KubeFedProductName} Operator
+
+You can install the {KubeFedProductName} Operator in the host cluster by following the {product-title} instructions on installing an Operator.
+
+For details, see the {product-title} documentation on xref:../applications/operators/olm-adding-operators-to-cluster.adoc[adding Operators to a cluster].
+
+include::modules/creating-kubefed-CRs.adoc[leveloffset=+1]
+
+== Deleting the {KubeFedProductName} Operator
+
+You can remove the {KubeFedProductName} Operator from the host cluster by following the {product-title} instructions on deleting an Operator.
+
+For details, see the {product-title} documentation on xref:../applications/operators/olm-deleting-operators-from-cluster.adoc[deleting Operators from a cluster].
```

Comment:
Same re: Additional Resources.

Reply:
I'd prefer to keep them as they are unless that's a real issue? This is the assembly level (can't have xrefs in modules anyway), it just has some headings and context lines around the steps rather than just using an include command?

Reply:
Oops, my b! Thought it was module. Carry on!